Formula/aescrypt.rb

```diff
@@ -19,13 +19,10 @@ def install
 
   test do
     (testpath/"key").write "kk=12345678901234567890123456789abc0"
-
-    require "open3"
-    Open3.popen3("#{bin}/aescrypt", "-k", testpath/"key") do |stdin, stdout, _|
-      stdin.write("hello")
-      stdin.close
-      # we can't predict the output
-      stdout.read.length > 0
-    end
+    original_text = "hello"
+    cipher_text = pipe_output("echo #{original_text} | #{bin}/aescrypt -k #{testpath}/key -s 128").chomp
```

Comment:
`pipe_output` is here to avoid using `echo` and having to deal with possible problems due to quoting etc. For that you have to use it as `pipe_output(<command>, <input>)`, or to be more specific:

```ruby
cipher_text = pipe_output("#{bin}/aescrypt -k #{testpath}/key -s 128", original_text)
```

Is the `.chomp` at the end really needed? Note that this will produce a slightly different result as the string `hello` is passed without a newline that is automatically appended by `echo`.